modules/FindBasisUniversal.cmake

```diff
@@ -115,100 +73,162 @@ macro(_basis_setup_source_file source)
     endif()
 endmacro()
 
+include(FindPackageHandleStandardArgs)
 # Find components
 foreach(_component ${BasisUniversal_FIND_COMPONENTS})
     if(_component STREQUAL "Encoder")
         if(NOT TARGET BasisUniversal::Encoder)
-            find_path(BasisUniversalEncoder_DIR NAMES basisu_frontend.cpp
-                HINTS "${BASIS_UNIVERSAL_DIR}" "${BASIS_UNIVERSAL_DIR}/encoder"
-                NO_CMAKE_FIND_ROOT_PATH)
-            if(BasisUniversalEncoder_DIR)
-                set(BasisUniversal_Encoder_FOUND TRUE)
+            # Try to find compiled library, e.g. when installed through vcpkg
```

Comment:
It's not `e.g.`, but rather *specific* to vcpkg -- there's no other case where this would be true.

Except ... if jherico's fork would be built and installed manually. But in that case, the two `if()`s below should check for debug `OR` release libs, as one can build & install just one of them.

Reply:
Well, should I create my own cmake project on basis, then that is another example for when this would be used.

Reply:
Yeah, then the `OR` note applies as well. Can you do this change, then (and then wrapping the `set_property()` in appropriate `if()`s as well)?